### PR TITLE
Cloudwatch: ListMetrics API page limit (#31788)

### PR DESCRIPTION
**Pull request by sunker**
* add list metrics api page limit

* Update docs/sources/datasources/cloudwatch.md

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>
(cherry picked from commit d512c5a1b49e295c1e83717c35ee77dc3c7661c3)

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sunker** [comment]
Had merge conflicts with 7.5.x so had to do a manual backport

File: docs/sources/datasources/cloudwatch.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ Filters syntax:
 Example `ec2_instance_attribute()` query
 
 ```javascript
-ec2_instance_attribute(us-east-1, InstanceId, { "tag:Environment": ["production"] });
+ec2_instance_attribute(us - east - 1, InstanceId, { 'tag:Environment': ['production'] });
```
Comment (dsotirakis) [inline docs/sources/datasources/cloudwatch.md:305]: Unnecessary spaces?

File: docs/sources/datasources/cloudwatch.md [hunk 2]
```diff
@@ -343,7 +343,7 @@ Tags can be selected by prepending the tag name with `Tags.`
 Example `ec2_instance_attribute()` query
 
 ```javascript
-ec2_instance_attribute(us-east-1, Tags.Name, { "tag:Team": ["sysops"] });
+ec2_instance_attribute(us - east - 1, Tags.Name, { 'tag:Team': ['sysops'] });
```
Comment (dsotirakis) [inline docs/sources/datasources/cloudwatch.md:346]: Unnecessary spaces?

**Review by dsotirakis — APPROVED** [review]
LGTM! (check comments on docs!)

**Pull request merged**